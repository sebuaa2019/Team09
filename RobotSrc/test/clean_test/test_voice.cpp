#include <ros/ros.h>
#include <std_msgs/String.h>

void voiceCB(const std_msgs::String::ConstPtr &msg){
    ROS_INFO("[test_voice]: %s\n", msg->data.c_str());
}

int main(int argc, char** argv){
    ros::init(argc, argv, "test_voice");
    ros::NodeHandle n;
    ros::Subscriber sub_sr = n.subscribe("/xfyun/iat", 10, voiceCB);

    ros::spin();

    return 0;
}
