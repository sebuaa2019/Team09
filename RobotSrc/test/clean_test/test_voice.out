[test_voice]: Stop
[test_voice]: Go
